pkg/job/delete_all.go

```diff
@@ -0,0 +1,49 @@
+package job
```

Comment:
@amitbhatt818 One of the first things that I would do is to explain what this PR is all about in the PR body.
It should explain everything to a reviewer to let this reviewer help as much as possible.

Can you rebase from master?

Reply:
Added description in the PR body. And rebased the branch.